src/facebookautomation.ts

```diff
@@ -0,0 +1,127 @@
+import puppeteer from "puppeteer";
+
```

Comment:
Always write file names splitted by dash (-) for example this filename has to be facebook-automation.ts